packages/neuron-wallet/package.json

```diff
@@ -14,7 +14,7 @@
   "license": "MIT",
   "scripts": {
     "start": "cross-env electron .",
-    "start:dev": "tsc && cross-env electron .",
+    "start:dev": "yarn clean && tsc && cross-env electron .",
     "build": "tsc",
     "clean": "rm -rf dist/*",
```

Comment:
does `rm -rf` works on windows?

Reply:
now we just think about macOS and Linux.

Reply:
Only **package** on macOS and linux, support development on windows

Reply:
👌, I try it.